src/cpu/cgroup_v2.rs

```diff
@@ -64,29 +88,45 @@ mod test {
     use std::path::Path;
 
     #[test]
-    fn test_read_v2_sys_measurement() {
-        let measurement =
-            read_and_parse_v2_sys_stat(&Path::new("fixtures/linux/sys/fs/cgroup_v2/cpu.stat_1"))
-                .unwrap();
+    fn test_read_v2_sys_measurement_default_cpu_max() {
+        let measurement = read_and_parse_v2_sys_stat(
+            &Path::new("fixtures/linux/sys/fs/cgroup_v2/cpu.stat_1"),
+            &Path::new("fixtures/linux/sys/fs/cgroup_v2/cpu.max_default"),
+        )
+        .unwrap();
         let cpu = measurement.stat;
         assert_eq!(cpu.total_usage, 171462000);
         assert_eq!(cpu.user, 53792000);
         assert_eq!(cpu.system, 117670000);
     }
 
+    #[test]
+    fn test_read_v2_sys_measurement_2_cpus() {
+        let measurement = read_and_parse_v2_sys_stat(
+            &Path::new("fixtures/linux/sys/fs/cgroup_v2/cpu.stat_1"),
+            &Path::new("fixtures/linux/sys/fs/cgroup_v2/cpu.max_2_cpus"),
+        )
+        .unwrap();
+        let cpu = measurement.stat;
+        assert_eq!(cpu.total_usage, 85731000);
+        assert_eq!(cpu.user, 53792000);
+        assert_eq!(cpu.system, 117670000);
+    }
+
     #[test]
     fn test_read_v2_sys_wrong_path() {
-        match read_and_parse_v2_sys_stat(&Path::new("bananas")) {
+        match read_and_parse_v2_sys_stat(&Path::new("bananas"), &Path::new("potato")) {
             Err(ProbeError::IO(_, _)) => (),
             r => panic!("Unexpected result: {:?}", r),
         }
     }
```

Comment:
```suggestion
    }
    
    #[test]
    fn test_read_v2_sys_wrong_path_for_max() {
        match read_and_parse_v2_sys_stat(&Path::new("fixtures/linux/sys/fs/cgroup_v2/cpu.stat_1"), &Path::new("potato")) {
            Err(ProbeError::IO(_, _)) => (),
            r => panic!("Unexpected result: {:?}", r),
        }
    }
```

I would add another test for when only the max file doesn't exist. Make sure it returns the expected error too.

Reply:
Do we want it to fail if the max file doesn't exist? We can still provide valuable data if it's not present.

Reply:
Oh yeah it should fall back on the default behavior not return an error here.